### PR TITLE
feature/flow

### PR DESCRIPTION
**Pull request by tkomstadius**
Added flow to the system.

### CODE REVIEW

**Comment by nattravn** [comment]
I think we can remove the link-list in /bruse_files:
Add more files
Add more files
Search File

**Comment by tkomstadius** [comment]
removed the links in bruse_files and added that page to the header links

**Comment by tkomstadius** [comment]
Maybe we should do the search page into a template that is connected with the help of a directive instead in the future.

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -12,6 +12,16 @@
   <%= content_tag :div, value, class: "flash #{key}" %>
 <% end %>
 
+<% if current_user %>
+  <%= link_to 'Bruse', root_url %> 
+  <% current_user.identities.each do |id| %>
+      <%= link_to 'My files', bruse_files_path(identity_id: id.id) %>
```
Comment (tkomstadius) [inline app/views/layouts/application.html.erb]: should the files be reached by the identity_id?

**Comment by nattravn** [comment]
it's nice to have this code merged. We can discuss later about the flow. So I say merge!

**Comment by klaseskilson** [comment]
Added the Identity name to the navigation.

**Pull request merged**